### ISSUE TITLE
Add more navigational landmarks and headings to the UI

### ISSUE THREAD

**Issue by MarcoZehe** [threema-ch/threema-web#621]
## Expected Behavior

Screen reader users should be able to move to more sections of the web UI via landmarks or heading navigation.

## Current Behavior

Except for the navigation landmark for the tabs, the UI is relatively flat, no way to easily find where the main contents starts after the messages list or the contacts, for example.

## Possible Solution

Adding a "main" landmark role and giving the conversation header text a heading, and also adding a blockquote role to quoted text in a message, and adding another heading for the start of unread messages makes the web UI much more navigable for screen reader users.

Pull request coming up.

## Steps to Reproduce (for bugs)

1. Load Threema Web with a screen reader.
2. Try to easily find the start of the main conversation area.
3. Try to find the first unread message in a chat. Currently the only, and rather ineffective, way is by searching for the string "unread".
4. Try to discern in a quoted message what is the quote and what the reply.

## Your Environment

* Threema Web version: 2.1-beta6
* Threema App version: 4.0 latest TestFlight
* Browser name and version: Firefox 63
* Computer operating system and version: Windows 10
* Smartphone operating system and version: iOS 12.0.1

**Comment by dbrgn** [comment]
Thanks @MarcoZehe, looking forward to the pull request!
